src/input/keyboard.c

```diff
@@ -104,8 +104,8 @@ match_keybinding_for_sym(struct server *server, uint32_t modifiers,
 			continue;
 		}
 		if (server->seat.nr_inhibited_keybind_views
-				&& server->focused_view
-				&& server->focused_view->inhibits_keybinds
+				&& server->active_view
+				&& server->active_view->inhibits_keybinds
```

Comment:
There may be a corner-case here: if a view (game, VM, etc.) is inhibiting keybindings and a layer-shell surface receives focus, the keybindings will still be inhibited. Probably not a big deal.

Reply:
We could add `&& view->surface == server.seat->keyboard_state.focused_surface`

Or wrap it in a `view_has_keyboard_focus()`

Reply:
I'm not sure if `view->surface == server.seat->keyboard_state.focused_surface` is quite correct because the view may have multiple "related" surfaces (for example xwayland-unmanaged popups).

Maybe to avoid complicating things too much, we just let this corner case be (i.e. just do the simple `focused_view` -> `active_view` replacement like you have currently).

Reply:
Yes, good point. If it becomes a problem, we'll have to think about it a bit deeper.